dbms/src/Flash/Planner/Plans/PhysicalExpand2.cpp

```diff
@@ -53,17 +53,46 @@ PhysicalPlanNodePtr PhysicalExpand2::build(
     ExpressionActionsPtrVec expression_actions_ptr_vec;
 
     // pre-detect the nullability change action in the first projection level and generate the pre-actions.
+    // for rollup, it's level projections may show like below:
```

Comment:
Is it possible to add test case in fullstack test?

Reply:
great!

Reply:
cool!